src/main/java/com/github/forax/umldoc/gen/Generator.java

```diff
@@ -36,5 +37,5 @@ void generate(boolean header, List<Entity> entities, List<Dependency> dependenci
    * @throws IOException if an I/O exception occurs
    */
   void generateSequenceDiagram(boolean header, Entity entryEntity, Method entryPoint,
-                               Writer writer) throws IOException;
+                               Set<String> entities, Writer writer) throws IOException;
```

Comment:
Hum, i think a wildcard is missing here

Reply:
Do you mean by adding **Set<? extends String>** ?
If so, I'm not sure to understand why because I just need to check if a String is present in a Set of String

Reply:
nope `Set<? super String` given
> I just need to check if a String is present in a Set of String